lib/checkbufferoverrun.cpp

```diff
@@ -508,8 +508,8 @@ ValueFlow::Value CheckBufferOverrun::getBufferSize(const Token *bufTok) const
     if (!var)
         return ValueFlow::Value(-1);
 
-    MathLib::bigint dim = std::accumulate(var->dimensions().begin(), var->dimensions().end(), 1LL, [](MathLib::bigint i1, const Dimension &dim) {
-        return i1 * dim.num;
+    MathLib::bigint dim = std::accumulate(var->dimensions().begin(),var->dimensions().end(),1LL,[](MathLib::bigint i1, const Dimension &paramDim) {
+        return i1 * paramDim.num;
```

Comment:
Is this a Cppcheck false negative also? Could you temporarily remove the false negatives from this PR so it can be solved properly separately..